### ISSUE TITLE
[Cura 4.x] Tree Supports Only Built Within Small Base Area

### ISSUE THREAD

**Issue by Enverex** [Ultimaker/Cura#5315]
**Application Version**
GIT Master (4.x)

**Platform**
Linux x86_64

**Printer**
Monoprice Ultimate / Duplicator 6

**Steps to Reproduce**
Add a large model that has big overhanging areas outside of the main mass. Enable tree supports.

**Actual Results**
Tree supports appear to only start from within the (rather small) skirt area on the base rather than being positioned whenever they're needed.

**Expected results**
Tree supports should start from wherever they're needed on the baseplate, which is the same behaviour as normal supports.

**Additional Information**
Here's a model with a large overhang that needs support. Note the lack of supports on the wings and that no supports appear outside of the zone within the skirt:
![image](https://user-images.githubusercontent.com/958670/52858536-523eb080-3122-11e9-9aac-f270fc109e90.png)

Increasing the allowed tree support angle and making the model a little smaller results in a bit of support being created, but only at an extreme angle from the base tree, rather than creating more trees on the build-plate to support areas that need support:
![image](https://user-images.githubusercontent.com/958670/52858582-726e6f80-3122-11e9-8441-8408de21cb3e.png)

Turning on normal supports works as you'd expect. It builds them wherever it needs them on the base plate:
![image](https://user-images.githubusercontent.com/958670/52858658-a77ac200-3122-11e9-9d1c-2461c5230c1f.png)

I can't find any option that explains why tree supports aren't being created in more sane locations.

**Comment by Ghostkeeper** [comment]
If you use normal support generation but then disable the "Use Towers" option, does it then similarly not create ordinary support?

**Comment by Ghostkeeper** [comment]
I found your dragon on Thingiverse, but for me it does create proper support under the wings.

![screenshot from 2019-02-18 11-25-16](https://user-images.githubusercontent.com/2448634/52944556-f5d0d080-336f-11e9-9d57-7d57b5c1b56c.png)

Maybe this has been fixed in 4.0 then? Or maybe it's just because I'm using different settings?

**Comment by Ghostkeeper** [comment]
Looking at your second screenshot, it appears that you should reduce the Tree Support Branch Angle setting. That will make the angle you describe less "extreme".

**Comment by Enverex** [comment]
This may have been an odd setting hidden away somewhere that I'd not reset and didn't realise because setting the angle to the default 40 still had the same issue.

I've cleared everything and reset my profiles, then loaded it again and now I'm getting the same as your image. I can't even recreate my original issue with any combination I've tried so far, so god knows which option was actually causing that.

Looks like this was a weird combination of options rather than a bug, sorry about that.
